src/guide/jbs-jdk-bug-system.md

```diff
@@ -46,7 +46,7 @@ A few things to keep in mind when filing an issue:
   * If you find a similar issue that was closed as [Cannot Reproduce]{.jbs-value} then it may be appropriate to re-open that one - if you don't have direct access to JBS you can file a bug using the [Bug Report Tool](https://bugreport.java.com/) requesting that it be reopened.
 * Make a reasonable attempt to narrow down which build or release the failure first appeared in.
 * Set [Affects Version/s]{.jbs-field} to the earliest JDK version(s) where the failure was seen.
-  * If the failure is found in an update train of the JDK (e.g. 11.0.x), if possible please see if it's also present in [mainline](https://github.com/openjdk/jdk).
+  * If the failure is found in an update train of the JDK (e.g. 11.0.x), please see (if possible) if it's also present in [mainline](https://github.com/openjdk/jdk).
```

Comment:
Not sure the proposed change makes any grammatical difference. But the bigger question is how they should reflect any findings on mainline in the "affects version" field? This is case where I would definitely want to see the latest version known to fail as well as the earliest.

Reply:
Added a sentence to indicate that adding the mainline version may be a good idea.